### DISCUSSION TITLE
Graphiql:true not working  - {"errors":[{"message":"Missing query"}]}

### DISCUSSION THREAD

**Discussion by faisvsqrl** [graphql/graphql-http#87] [Q&A] [answered]
Created new fresh project followed steps from provided documentation and I set graphiql:true and hit localhost:3001/graphql from browser so it's just showing following error instead graphqlUI.

{"errors":[{"message":"Missing query"}]}

`const express = require('express');
const app = express();
const port = 3001;
const { createHandler } = require('graphql-http/lib/use/express');

const { GraphQLSchema, GraphQLObjectType, GraphQLString } = require('graphql');

const schema = new GraphQLSchema({
  query: new GraphQLObjectType({
    name: 'Query',
    fields: {
      hello: {
        type: GraphQLString,
        resolve: () => 'world',
      },
    },
  }),
});

app.get('/', (req, res) => {
  res.send('Hello World!')
})

// Create a express instance serving all methods on `/graphql`
app.all('/graphql', createHandler({ schema,graphiql:true }));

app.listen(port, () => {
  console.log(`Example app listening on port ${port}`)
})`

**Comment by enisdenjo** [comment, marked as answer]
This library implements only the GraphQL over HTTP spec, please see the [disclaimer in the readme](https://github.com/graphql/graphql-http#only-graphql-over-http).

Also see #38.

**Reply by faisvsqrl** [reply]
@enisdenjo Add more details so user can easily understand, like 

without any additional features (like playground, UI Interface, file uploads, @stream/@defer directives and subscriptions).

**Reply by enisdenjo** [reply]
A PR is very welcome! 😄

**Reply by enisdenjo** [reply]
https://github.com/graphql/graphql-http/commit/fe135497e9428486609048a0a43089dd8606b800
